### PR TITLE
Add ShortShipReturnInvoice

### PR DESCRIPTION
**Pull request by jordan-brough**
For handling short ship "refunds".

### CODE REVIEW

File: app/models/spree_avatax/short_ship_return_invoice.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      }
+    end
+
+    def gettax_line_params(unit_cancels:, taxed_at:)
+      unit_cancels.map do |unit_cancel|
+        inventory_unit = unit_cancel.inventory_unit
+
+        adjustment = unit_cancel.adjustment
+
+        if adjustment
```
Comment (philbirt) [inline app/models/spree_avatax/short_ship_return_invoice.rb]: Nice. should we add a todo to remove this? Or do you think its worthwhile keeping it here forever?

File: spec/models/spree_avatax/short_ship_return_invoice_spec.rb [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+      let(:unit_cancels2) do
+        Spree::OrderCancellations.new(order2).short_ship(order2.inventory_units)
+      end
+
+      it 'fails' do
+        expect {
+          SpreeAvatax::ShortShipReturnInvoice.generate(
+            unit_cancels: unit_cancels1 + unit_cancels2
+          )
+        }.to raise_error(/more than one order/)
```
Comment (philbirt) [inline spec/models/spree_avatax/short_ship_return_invoice_spec.rb]: Nice

**Comment by philbirt** [comment]
Awesome work :+1:

File: app/models/spree_avatax/short_ship_return_invoice.rb (hunk 1 above)
Comment (jordan-brough) [inline app/models/spree_avatax/short_ship_return_invoice.rb]: Good point, yeah I don't know.  I'll add a comment for now about possibly removing it.  After we get the capture stuff tidied up and out it'll be good to think about if there are any valid use cases for no-adjustment here.

**Pull request closed without merging**